proposals/0085-additional-fee-collector-constraints.md

```diff
@@ -0,0 +1,61 @@
+---
+simd: "0085"
+title: Additional Fee Collector Constraints
+authors:
+  - Justin Starry
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-05
+feature: https://github.com/solana-labs/solana/issues/33888
+---
+
+## Summary
+
+Burns fees if they would otherwise be collected into an account that violates
+one of the following constraints:
+
+1. Must be a system owned account
+2. Must be rent-exempt
+
+This applies to both transaction fees and rent fees. Note that the "rent-exempt"
+constraint was already added for rent fee collection in
+[Feature 30151](https://github.com/solana-labs/solana/issues/30151).
+
+## Motivation
+
+1. Since fee collection occurs outside of the runtime, it's generally a good
```

Comment:
How is it that fee collection occurs outside of the runtime? This seems like an implementation detail.

Reply:
Fees are deposited at the end of the block and must be done consistently across clients to maintain consensus. By outside the runtime, I meant outside of normal transaction processing.